fastchat/utils.py

```diff
@@ -349,3 +350,30 @@ def str_to_torch_dtype(dtype: str):
         return torch.bfloat16
     else:
         raise ValueError(f"Unrecognized dtype: {dtype}")
+
+
+def load_image(image_file):
+    from PIL import Image
+    import base64
+
+    image = None
+
+    if image_file.startswith("http://") or image_file.startswith("https://"):
+        response = requests.get(image_file)
+        image = Image.open(BytesIO(response.content)).convert("RGB")
+    elif base64.b64encode(base64.b64decode(image_file)) == image_file.encode():
```

Comment:
If the input is not in the base64 format, this line will raise an error. Maybe swap the last two branches, or catch the error?